ui/shared/utils/constants.js

```diff
@@ -1316,37 +1316,43 @@ export const SV_IN_SILICO_GROUP = 'Structural'
 export const NO_SV_IN_SILICO_GROUPS = [MISSENSE_IN_SILICO_GROUP, CODING_IN_SILICO_GROUP]
 export const SPLICE_AI_FIELD = 'splice_ai'
 
+export const PRED_COLOR_MAP = ['green', '#90ee90', 'grey', 'yellow', 'red', '#8b0000']
```

Comment:
I changed to use the `ColoredIcon` component, which doesn't support the `olive` color. So I keep the light green color encoded.

Reply:
So we do not want to change to that, that will change all the main colors to be css colors and not semantic UI colors, and as is discussed extensively in the ticket, we are trying to keep using the sematic ui colors wherever possible. You need to add conditional logic for which icon component to use, and only use the custom color for dark red